### PR TITLE
Publish v1.15. [release]

### PR DESCRIPTION
**Pull request by ryanbourdais**
Publish v1.15. [release]

### CODE REVIEW

**Comment by ryanbourdais** [comment]
As mentioned in #119 elixir 1.15 is out, this PR creates the images for this new version and should close #119

File: 1.15/23.3.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# October.
+
+# Erlang version 23.3 does not support OpenSSL3, which means only for this specific version do we need to use 20.04. Please adjust accordingly
+
+FROM cimg/base:2023.04-20.04
+
+LABEL maintainer="Community & Partner Engineering Team <community-partner@circleci.com>"
+
+# Install Erlang via Erlang Solutions' .deb
+ENV ERLANG_VERSION="23.3.1"
```
Comment (JalexChen) [inline 1.15/23.3.1/Dockerfile:16]: i would check to see if there are newer erlang versions, and whether they are compatible

**Review by JalexChen — CHANGES_REQUESTED** [review]

File: 1.15/23.3.1/Dockerfile (hunk 1 above)
Comment (ryanbourdais) [reply 1.15/23.3.1/Dockerfile:16]: fixed :+1:

**Comment by JalexChen** [comment]
what do you think about keeping erlang 23 in the manifest, but removing the directory associated with 1.15?

**Comment by ryanbourdais** [comment]
@JalexChen just re-added it to the manifest, its gonna be a bit awkward to update this image until Erlang 26 releases but hopefully that will be soon.

**Review by JalexChen — APPROVED** [review]

**Pull request merged**